index.bs

```diff
@@ -87,20 +87,20 @@ interface VRDisplay : EventTarget {
   readonly attribute DOMString displayName;
 
   /**
+   * Populates the passed VRFrameData with the information required to render
+   * the current frame.
+   */
+  void getCurrentFrameData(VRFrameData frameData);
```

Comment:
The more code and docs I write about this the more this function name feels overly verbose to me. Anyone object to simply `getFrameData()`?

Also, I think it should return a bool to indicate whether or not the frame data was populated.